me.gladwell.eclipse.m2e.android/src/main/java/me/gladwell/eclipse/m2e/android/configuration/AttachDocumentsClasspathLoader.java

```diff
@@ -29,7 +30,7 @@ public AttachDocumentsClasspathLoader(@PrunePlatformProvidedDependencies Classpa
     }
 
     @Override
-    public Iterable<IClasspathEntry> load(IJavaProject project) throws FileNotFoundException {
+    public Iterable<IClasspathEntry> load(IJavaProject project) throws FileNotFoundException, CoreException {
```

Comment:
Yeah that is the third option, i forgot to write down. Thanks, I will use that way.